web/src/infra/gameInfo/GameInfo.tsx

```diff
@@ -89,7 +89,10 @@ class GameInfo extends React.Component<gameInfoProps, {}> {
     if (!GAMES_MAP[gameCode] && ctx.res) {
       return generatePageError(404);
     }
-    return { gameCode };
+    return {
+      gameCode,
+      namespacesRequired: ['common', 'GameModePicker'],
```

Comment:
Isn't this supposed to be "GameInfo" given that this is another page?  Was this changed intended ?

Reply:
I thought we needed to specify each namespace required by inner components, but I need to read the docs again to see if it's really necessary. Can I do that tomorrow? :)

Reply:
I am reverting these changes before merging

Reply:
Nevermind! This is WAI, I got this error when I removed:
```
You have not declared a namespacesRequired array on your page-level component: withRouter(GameInfo). This will cause all namespaces to be sent down to the client, possibly negatively impacting the performance of your app. For more info, see: https://github.com/isaachinman/next-i18next#4-declaring-namespace-dependencies
```

Reply:
You were right :), we need to specify the namespace

Reply:
Yeah! I don't like to have to specify it twice, but it takes advantage of SSR to speed up things. 

https://github.com/isaachinman/next-i18next/tree/v7.0.1#4-declaring-namespace-dependencies

> The `withTranslation` HOC is responsible for passing the `t` function to your component. It enables all the translation functionality provided by `i18next`. Further, it asserts your component gets re-rendered on language change or changes to the translation catalog itself (loaded translations). More info can be found [here](https://react.i18next.com/latest/withtranslation-hoc).

> By default, `next-i18next` will send _all your namespaces_ down to the client on each initial request. This can be an appropriate approach for smaller apps with less content, but a lot of apps will benefit from splitting namespaces based on route.

> To do that, you need to return a `namespacesRequired` array via `getInitialProps` on your page-level component. You can see this approach in [examples/simple/pages/index.js](./examples/simple/pages/index.js).

> Note: `withTranslation` provides namespaces to the component that it wraps. However, `namespacesRequired` provides the total available namespaces to the entire React tree and belongs on the page level. Both are required (although you can use `Trans` instead of `withTranslation` if desired).